interface/forms/clinical_instructions/new.php

```diff
@@ -30,31 +30,29 @@
 
```

Comment:
Also is the same issue here.
![image](https://user-images.githubusercontent.com/46497068/91076960-4765b580-e60e-11ea-8592-ee46a3669023.png)